CHANGELOG.md

```diff
@@ -52,6 +52,9 @@ Bug fixes:
 - [#243](https://github.com/HewlettPackard/oneview-chef/issues/243) Chef-client 13.2.20 does not allow modification of property :save_resource_info
 - [#180](https://github.com/HewlettPackard/oneview-chef/issues/180) Create Mixins for the resource providers common methods
 
+### Breaking changes
+- The `new_profile` action of oneview_server_profile_template was removed.
```

Comment:
I suggest saying that it was "permanently replaced by" the corresponding action.

Reply:
ok, but that action was not replaced in fact. 
What do you think if we use this below text?
"The `new_profile` action of oneview_server_profile_template was permanently removed. Instead, use the `create` action of server_profile setting the server_profile_template property."

Reply:
The `:new_profile` action was permanently replaced by specifying a `server_profile_template` on creates and updates of the `server_profile` resource. They should now behave

Reply:
Perhaps better:
The `:new_profile` action was incorporated into create and update actions of the `server_profile` resource where a `server_profile_template` is specified.